packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -210,7 +210,6 @@ _Props/Attrs_
     -   `setter` - this will parse the markdown string into a Node using
```

Comment:
Assuming we stick with the current approach, could you update since it's not a getter/setter any more?

Reply:
Updated the spec.